src/library/scala/math/PartialOrdering.scala

```diff
@@ -85,3 +85,9 @@ trait PartialOrdering[T] extends Equiv[T] {
     override def equiv(x: T, y: T) = outer.equiv(y, x)
   }
 }
+
+object PartialOrdering {
+  implicit def partialOrderingFromOrdering[T](implicit ev: Ordering[T]): PartialOrdering[T] = ev
+
+  @inline def apply[T](implicit ev: PartialOrdering[T]): Equiv[T] = ev
```

Comment:
I would think this should return `PartialOrdering[T]`

Reply:
```suggestion
  @inline def apply[T](implicit ev: PartialOrdering[T]): PartialOrdering[T] = ev
```

Reply:
Ugh sloppy copy and paste error. And the universal instance masked the error. I'll fix this and improve the unit tests to make sure that they are resolving the expected instance.

Thank you; good catch.